Cargo.toml

```diff
@@ -88,19 +88,19 @@ ahash = { version = "0.8", default-features = false, features = [
     "runtime-rng",
 ] }
 apache-avro = { version = "0.17", default-features = false }
-arrow = { version = "55.1.0", features = [
+arrow = { path = "/Users/zhuqi/arrow-rs/arrow", features = [
```

Comment:
If you want to experiment, need to change this to the branch:

https://github.com/apache/arrow-rs/pull/7624